jscomp/core/js_implementation.mli

```diff
@@ -37,3 +37,8 @@ val implementation :
 
 val implementation_cmj : Format.formatter -> string -> unit
 (** [implementation_cmj ppf cmj_file] compiles a cmj to JS *)
+
+module Ppx_entry : sig
+  val rewrite_signature : Parsetree.signature -> Parsetree.signature
```

Comment:
you probably found out that this doesn't do what you think it does. 

You can revert this change, it doesn't need to be exposed.

Reply:
removed in https://github.com/melange-re/melange/pull/538/commits/abdb6ae192772eb3bbd90c280c055e3977d32fbc.